src/link/utils.js

```diff
@@ -1,3 +1,16 @@
 export const getSiteLanguage = () => {
 	return document.documentElement.getAttribute( 'lang' ).split( '-' )[ 0 ];
 };
+
+export const isTextNearTheEdge = ( anchor ) => {
+	const anchorXPosition = anchor.getBoundingClientRect();
+	const scrollWidth = document.body.scrollWidth;
+	if (
+		anchorXPosition.left / scrollWidth < 0.2 ||
```

Comment:
You could return this expression directly instead of the `return true` / `return false`.